lib/ansible/modules/storage/netapp/na_ontap_volume.py

```diff
@@ -222,6 +222,17 @@
     - zh_tw.big5        Traditional Chinese Big 5
     - To use UTF-8 as the NFS character set, append '.UTF-8' to the language code
     version_added: '2.8'
+
+  qos_policy_group:
+    description:
+    - Allows a QoS policy group to be set on volume creation.
```

Comment:
is this a bool or permission? If it is, perfect but if not I would say `Specifies` rather than `Allows`.

Reply:
could you possibly add examples in as well for docs?

Reply:
updated to 'Specifies' and included in example